guides/plugins/apps/app-base-guide.md

```diff
@@ -229,7 +259,7 @@ from the Admin API.
 You can find out more about how to use these credentials in our Admin API authentication guide:
 
 <!-- markdown-link-check-disable-next-line -->
-{% embed url="https://shopware.stoplight.io/docs/admin-api/ZG9jOjEwODA3NjQx-authentication-and-authorisation#integration-client-credentials-grant-type" caption="Admin API Authentication & Authorisation" %}
+{% embed url="<https://shopware.stoplight.io/docs/admin-api/ZG9jOjEwODA3NjQx-authentication-and-authorisation#integration-client-credentials-grant-type>" caption="Admin API Authentication & Authorisation" %}
```

Comment:
This might break gitbook, and it will definitely break our migration script. Why the < and >?